theano/gof/graph.py

```diff
@@ -125,8 +125,12 @@ def run_params(self):
         Returns the params for the node, or NoParams if no params is set.
 
         """
-        if hasattr(self.op, 'get_params'):
-            return self.op.get_params(self)
+        if hasattr(self.op, 'params_type'):
```

Comment:
Ok. So, now that I have added a default get_params() implementation, I should better check here if the op also has params_type, right ?